nrfutil/device/logLibVersions.ts

```diff
@@ -108,15 +108,19 @@ export default async (moduleVersion: ModuleVersion) => {
                     `It looks like you have installed JLink using ${JLinkInstallerVersion}, but currently we only support their Universal Installer for your system.`
                 );
                 logger?.warn(
-                    `Please install JLink: https://www.segger.com/downloads/jlink/JLink_MacOSX_V788j_universal.pkg`
+                    `Please install JLink: https://www.segger.com/downloads/jlink/JLink_MacOSX_V794e_universal.pkg`
                 );
             }
         }
     } catch (error) {
-        logger?.error(
-            `Failed to get the library versions${
-                error == null ? `: ${describeError(error)}` : ''
-            }`
-        );
+        let message = '';
+        if (error == null) {
+            message = describeError(error);
+        }
+        if (error === undefined) {
+            message = '';
+        }
+
+        logger?.error(`Failed to get the library versions${message}`);
```

Comment:
```suggestion
       logger?.error(
             `Failed to get the library versions${
                 error != null ? `: ${describeError(error)}` : ''
             }`
         );
```

Reply:
didn't do that because it changes existing logic.
"regardless I don't understand why we use describeError(error) when error is null or undefined IMO it should be as the cmt below" - yes I too did not understand that but decided not to change.

null & undefined are handled in describeError() and converted to strings there, but that doesn't make sense in this use case